management_app/views/course.py

```diff
@@ -2,17 +2,18 @@
 from werkzeug.utils import secure_filename
 import pandas as pd
 from datetime import date
-import re
+import re, openpyxl, os
 
 from management_app.views.auth import login_required
 from management_app.db import get_db
 from management_app.views.utils import download_file, upload_file, remove_upload_file, get_upload_filepath, insert_log, convert_local_timezone
 from management_app.views.points import calculate_teaching_point_val, update_yearly_ending_balance, get_latest_academic_year
 
-
-
 courses = Blueprint('courses', __name__, url_prefix='/courses')
 
+BASE_DIR = os.path.realpath(os.path.join(os.path.dirname(__file__), '..'))
+DOWNLOAD_FOLDER = 'static/data_templates'
```

Comment:
Minor: these variables could be re-used from `utlis.py`.

Reply:
Fixed.